scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
```

Comment:
:stop_sign: **Security Vulnerability**: SQL injection vulnerability detected. Directly interpolating user input into SQL queries can lead to unauthorized data access or manipulation[^1]. Use parameterized queries or prepared statements instead of string concatenation.
```suggestion
// SQL injection mitigation
// Assuming you're using a library like 'mysql2' for MySQL or 'pg' for PostgreSQL
const query = 'SELECT * FROM users WHERE id = ?';
connection.query(query, [process.argv[2]], (error, results) => {
    if (error) throw error;
    console.log('User:', results[0]);
});
```

[^1]: CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection') - https://cwe.mitre.org/data/definitions/89.html